components/wmtk_components/regular_space/internal/RegularSpace.hpp

```diff
@@ -12,7 +12,7 @@ namespace wmtk::components::internal {
 class RegularSpace
 {
     TriMesh& m_mesh;
-    bool m_lock_boundary;
+    // bool m_lock_boundary;
```

Comment:
Why did you comment that out? Is it unused? In that case, please just open up an issue but do not change the code itself.